net_connection/src/json_protocol.rs

```diff
@@ -2,6 +2,8 @@
 //! basically handles serialization / deserialization from / to the core
 //! protocol message types (NamedBinary and Json).
 
+#![allow(non_snake_case)]
```

Comment:
why are we doing this? i don't see any non-snake-case variables in here. shouldn't we stick with default rust style stuff?

Reply:
Its for the unit test function names like `it_can_convert_HandleSendMessageResult()`
maybe I can move it in the test mod.